api/k8s/v1/k8s.proto

```diff
@@ -553,6 +553,11 @@ message ResizeHPARequest {
     uint32 max = 2;
   }
   Sizing sizing = 5 [ (validate.rules).message = {required : true} ];
+
+  Sizing old_sizing = 6;
+
+  // A flag to add sirens to the audit message or not
+  bool resize_warning = 7;
```

Comment:
Just curious, why have the client determine if a warning should be shown?

Reply:
Not really a good reason, but we can't do math inside the YAML file, so we would need to add some interception to the audit / aimbot functionality, which I figured was more involved than we wanted. Maybe I could just split things and have this PR just be the `current_sizing` and leave the warning for another?